Source/WebKit/UIProcess/API/Cocoa/_WKApplicationManifest.mm

```diff
@@ -77,6 +77,22 @@
     };
 }
 
+static std::optional<WebCore::ApplicationManifest::Shortcut> makeVectorElement(const WebCore::ApplicationManifest::Shortcut*, id arrayElement)
```

Comment:
Where is this function used?

Reply:
This is used later in line 259 in -initWithCoder::
`makeVector<WebCore::ApplicationManifest::Shortcut>(shortcuts),`

Implemented in VectorCocoa.h, `makeVector` is a template that calls into `makeVectorElement`.